src/examples/20-seamless.html

```diff
@@ -18,10 +18,10 @@
             The seamless skin uses the height of your textarea as the minimum
             editor height.
         -->
-        <textarea class="wymeditor" style="min-height: 600px"></textarea>
+        <textarea class="wymeditor" style="min-height: 300px"></textarea>
         <input type="submit" class="wymupdate" />
     </form>
-    <div class="more-content">The toolbar will stop floating here.<div>
+    <p style="height: 5000px">The floating toolbar will stop here.</p>
```

Comment:
Does the editor allow giving a `p` tag height? If not I would suggest we don't do that. Test/examples should be possible in the editor.